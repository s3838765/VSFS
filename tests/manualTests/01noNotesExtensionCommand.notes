NOTES V1.0
@file1
 This is some file data
@secondfile
 This is the second file data
 There are two lines here.
=dir1/
@dir1/filexyz
