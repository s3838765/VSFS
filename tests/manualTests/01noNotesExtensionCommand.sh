#! /usr/bin/sh
../../VSFS list 01noNotesExtensionCommand 1> 01noNotesExtensionCommand.out 2>&1
