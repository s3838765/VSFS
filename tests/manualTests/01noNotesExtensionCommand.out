drwxrwxrwx 1 dib dib 512 Oct 23 17:06 dir1/
-rwxrwxrwx 1 dib dib   0 Oct 23 17:06 dir1/filexyz
-rwxrwxrwx 1 dib dib  22 Oct 23 17:06 file1
-rwxrwxrwx 1 dib dib  53 Oct 23 17:06 secondfile
